examples/data_pipeline/roberta_datapipe.py

```diff
@@ -0,0 +1,76 @@
+from argparse import ArgumentParser
+from functools import partial
+from typing import Dict, Any
+
+import torchtext.functional as F
+import torchtext.transforms as T
+from torch.hub import load_state_dict_from_url
+from torch.nn import Module
+from torch.utils.data import DataLoader
+from torchtext.datasets import SST2
+
+
+class RobertaTransform(Module):
```

Comment:
This may not be necessary to do since this is an example but I did notice the `__init__` method for the `RobertaTransform` is the exact same for both datapipes and I was wondering if it would be possible to consolidate the transforms by creating a base class. The child class would then extend the base class and implement the forward method (which is what expects different inputs i.e. `DataFrame` vs a `Dict` object).

Totally fine if you think this is out of scope for a simple example but I was thinking it might make it more clear as to what the main differences are in the 2 `RobertaTransform` implementations.

Reply:
> the __init__ method for the RobertaTransform is the exact same for both datapipes and I was wondering if it would be possible to consolidate the transforms by creating a base class

Yes, that's true but only for now. As we move tokenization and vocabulary to work natively with TorchArrow (instead of operating as UDFs), the APIs and initialization would not be the same.

Reply:
Gotcha I missed that part. Thanks for providing that clarification!